test/test/helpers/use-fake-dom/index.js

```diff
@@ -25,8 +25,8 @@ export default function domWrapper( markup, features ) {
 
 domWrapper.withContainer = function withContainer() {
```

Comment:
I noticed all the test helpers are included in `react-test-env` too:

https://github.com/Automattic/react-test-env/blob/e60b7a4/lib/index.js#L59-L77

Should we simply re-export them here?